src/v4/PublicApi/Components/AmityEmptyNewsFeedComponent/Elements/ExploreCommunityButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { Image, Text, TouchableOpacity } from 'react-native';
+import React, { memo, useCallback } from 'react';
+import { ComponentID, ElementID, PageID } from '../../../../enum';
+import useConfig from '../../../../hook/useConfig';
+import { useConfigImageUri, useUiKitConfig } from '../../../../hook';
+import { useStyles } from './styles/styles';
+
+const ExploreCommunityButton = () => {
+  const { excludes } = useConfig();
+  const styles = useStyles();
+
+  const [text] = useUiKitConfig({
+    keys: ['text'],
+    page: PageID.social_home_page,
+    component: ComponentID.empty_newsfeed,
+    element: ElementID.explore_communities_button,
+  }) as string[];
+
+  const icon = useConfigImageUri({
+    configKey: 'icon',
+    configPath: {
+      page: PageID.social_home_page,
+      component: ComponentID.empty_newsfeed,
+      element: ElementID.explore_communities_button,
+    },
+  });
+
+  const onPressExploreCommunity = useCallback(() => {}, []);
```

Comment:
Should we remove this ?

Reply:
At that time, PDM and Stake holders does not provide the action of that function. I got the answer last night, I will implement it

Reply:
Updated !